dtcm/ascension/map.xml

```diff
@@ -0,0 +1,124 @@
+    <spawn team="red-team" kit="default-kit" yaw="-180" region="red-spawn"/>
+    <default region="obs-spawn"/>
+</spawns>
+<!-- Monuments -->
+<destroyables name="Wool" completion="22%" materials="wool" show-progress="true" sparks="true">
+    <destroyable owner="blue-team" region="blue-monument"/>
+    <destroyable owner="red-team" region="red-monument"/>
+</destroyables>
+<!-- Filters -->
+<filters>
+    <not id="blue-deny-explosions">
+        <all>
+            <team>blue-team</team>
+            <cause>explosion</cause>
+        </all>
+    </not>
+    <not id="red-deny-explosions">
+        <all>
+            <team>red-team</team>
+            <cause>explosion</cause>
+        </all>
+    </not>
+</filters>
+<!-- Regions -->
+<regions>
+    <!-- Definitions -->
+    <union id="spawns">
+        <point id="blue-spawn">34.5,19,28.5</point>
+        <point id="red-spawn">122.5,19,28.5</point>
+        <point id="obs-spawn">78.5,12.5,-33.5</point>
+    </union>
+    <union id="spawn-protections">
+        <cuboid id="blue-spawn-protection" min="31,6,20" max="38,24,31"/>
+        <cuboid id="red-spawn-protection" min="119,6,20" max="126,24,31"/>
+    </union>
+    <union id="monuments">
+        <cuboid id="blue-monument" min="33,12,-35" max="36,17,-32"/>
+        <cuboid id="red-monument" min="120,12,-35" max="124,17,-32"/>
+    </union>
+    <negative id="not-map">
+        <rectangle id="map" min="23,-40" max="134,29"/>
+    </negative>
+    <!-- Applications -->
+    <apply region="not-map" block="never" message="You may not edit outside the map!"/>
+    <apply region="spawn-protections" block="never" message="You may not edit spawn!"/>
+    <apply region="blue-spawn-protection" enter="blue-team" message="You may not enter the enemy's spawn!"/>
+    <apply region="red-spawn-protection" enter="red-team" message="You may not enter the enemy's spawn!"/>
+    <apply region="blue-monument" block="red-deny-explosions"/>
+    <apply region="red-monument" block="blue-deny-explosions"/>
+</regions>
+<!-- Item Remove -->
+<itemremove>
+    <item>iron helmet</item>
+    <item>chainmail chestplate</item>
+    <item>leather leggings</item>
+    <item>iron boots</item>
+    <item>wool</item>
+    <item>wood</item>
+    <item>arrow</item>
+</itemremove>
+<!-- Tool Repair -->
+<toolrepair>
+    <tool>iron sword</tool>
+    <tool>bow</tool>
+    <tool>iron pickaxe</tool>
+    <tool>iron axe</tool>
+</toolrepair>
+<itemkeep>
+    <item>golden apple</item>
+</itemkeep>
+<!-- Kill Reward -->
+<kill-rewards>
+    <kill-reward>
+        <item amount="8" material="arrow"/>
+        <item amount="1" material="golden apple"/>
+    </kill-reward>
+</kill-rewards>
+<crafting>
+    <disable>shears</disable>
+</crafting>
+<!-- Misc -->
+<maxbuildheight>26</maxbuildheight>
+<disabledamage>
+    <damage ally="true" self="false" enemy="false" other="false">block explosion</damage>
+</disabledamage>
+</map>
```

Comment:
The include also disables hunger, you'll need to readd that part since there's no food in the kit anymore.